department-of-veterans-affairs/vets-website
# Generate new form

## Summary
Add Income and Asset Statement Form (21P-0969)

### Description
Generate app and get general scaffolding up and running

## Issue
[Generate new vets-website application#81941](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81941)

### Related Issues
[Add feature toggle #81935](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81935)

### Epic
[Digitize Income and Asset Statement Form 21P-0969](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/80925)

## How to run in local environment
1. Check out this branch locally in 'vets-website'
2.  Checkout this [branch](https://github.com/department-of-veterans-affairs/content-build/pull/2075) locally in 'content-build'
3. Run 'vets-api' in your local environment
4. Run the 'vets-website' app with `yarn watch --env entry=static-pages,auth,login-page,profile,dashboard,income-and-asset-statement`

## How to verify
1. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser
2. Verify the 'Introduction' page is loaded
3. Start the application without signing in
4. Continue the application
5. Submit the application
6. Verify that the 'Confirmation' page displayed

## Testing done

## Screenshots
n/a

## What areas of the site does it impact?
Income and Asset Statement Application

### Quality Assurance & Testing

### Error Handling

### Authentication
